index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <header id="header">
+    <img src="images.png" alt="image logo " id="header-img" width="50px">
+    <nav id="nav-bar">
+      <a href="#features" class="nav-link">Features</a>
+      <a href="#how_it_works" class="nav-link">How It works</a>
+      <a href="#pricing" class="nav-link">Pricing</a>
+    </nav>
+
+  </header>
+  <h1>Hello World, we are software developers!</h1>
+
+  <section id="features">
+    <iframe id="video" width="300" height="300" src="https://www.youtube.com/embed/50kH47ZztHs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+  </section>
+  <section id="how_it_works">
+    <form id="form" action="https://www.freecodecamp.org/email-submit" method="post">
+      <input placeholder="Email" type="email" id="email" name="email" required>
+      <input id="submit" type="submit">
+    </form>
+  </section>
+  <section id="pricing">
+
+    <div id="premium">
+      <img style="width: 50px; height: 50px;" src="https://www.logolynx.com/images/logolynx/8e/8e8a0eb7ebb357a059c663585640e691.jpeg" alt="logo">
```

Comment:
kindly remove the style line from the HTML, the code can look better if you put the styles on the Css sheet.

Reply:
Nice here. You can say your code would look more semantic.